.github/workflows/lint.yml

```diff
@@ -24,8 +25,12 @@ jobs:
           chmod +x llvm.sh
           sudo ./llvm.sh 18
           sudo apt-get install -y --no-install-recommends clang-format-18
+          curl -fsSL https://deb.nodesource.com/setup_22.x | sudo bash
```

Comment:
Non-blocking: We use Node 20 everywhere else so far and that should be installed by default on the runner image already (https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#language-and-runtime), although I'm not sure if using it involves any additional steps.

Reply:
There is a `setup-node` github workflow maintained by Github itself. We should use the official distributions.